vgaidarji/github-android-app
# Migrate to compose navigation and remove fragments completely

### What has been done
- Removed screen fragments and replaced navigation with compose navigation. Implemented bottom navigation using compose
- Added Splash screen to navigate between screens and created nested navigation graph for main screens 
- Changed bottom navigation items selected/unselected colors

[Screen_recording_20240717_133538.webm](https://github.com/user-attachments/assets/a7ef95e7-344f-4acb-9c52-4ff0dc2a596e)

### How to test
- Launch the application and navigate between tabs. Test back navigation to make sure it handled properly (back navigation from main screen closes the app instead of navigating back between screens in navigation stack).
